### DISCUSSION TITLE
Error importin Addchanneld in Sagemaker

### DISCUSSION THREAD

**Discussion by jonathanquantumalpha** [Project-MONAI/MONAI#7129] [Q&A]
Hi I tried importing Addchanneld transforms into Amazon Sagemaker and it came out as error. Can you please help:

from monai.transforms import(
    Compose,
    AddChanneld,
    LoadImaged
)

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[31], line 10
      8 import numpy as np
      9 import nibabel as nib
---> 10 from monai.transforms import(
     11     Compose,
     12     AddChanneld,
     13     LoadImaged,
     14     Resized,
     15     ToTensord,
     16     Spacingd,
     17     Orientationd,
     18     ScaleIntensityRanged,
     19     CropForegroundd,
     20     EnsureChannelFirstd
     21 )
     22 from monai.data import DataLoader, Dataset, CacheDataset
     23 from monai.utils import set_determinism

ImportError: cannot import name 'AddChanneld' from 'monai.transforms' (/home/ec2-user/anaconda3/envs/pytorch_p310/lib/python3.10/site-packages/monai/transforms/__init__.py)

**Comment by KumoLiu** [comment]
Hi @jonathanquantumalpha, it was deprecated for MONAI v1.3 in https://github.com/Project-MONAI/MONAI/pull/7019. We would suggest using `EnsureChannelFirst`.

Thanks!
